docs/migrate/migrate-guide.rst

```diff
@@ -172,78 +214,6 @@ An expectation value of an observable could be the target quantity in
 scenarios where knowing a quantum state is not relevant. This
 often occurs in optimization problems or chemistry applications.  For example, when trying to discover the extremal energy of a system.
 
-.. raw:: html
-
-   </details>
-
-.. raw:: html
-
-  <details>
-  <summary>Which parts of my code do I need to refactor?</summary>
```

Comment:
Did you mean to delete this section?

Reply:
Yes - @ElePT thought we covered this thoroughly elsewhere.